src/mod/databases/mod_pgsql/mod_pgsql.c

```diff
@@ -779,6 +779,13 @@ switch_status_t pgsql_next_result_timed(switch_pgsql_handle_t *handle, switch_pg
 		case PGRES_PIPELINE_SYNC:
```

Comment:
shouldn't this be removed as its added back right below?  also seems that this doesn't require 14 if its already there so it shouldn't be in the #if